src/App.js

```diff
@@ -22,34 +22,59 @@ const App = () => {
   const [hand, setHand] = useState([])
 
   const drawCard = () => {
-    // 1)
+    // 1) this is just the name of the function, it is 'drawCard', followed by an arrow function and the curly brace that the code block is encapsulated in.
     const draw = `${rank[Math.floor(Math.random() * rank.length)]} ${
       suit[Math.floor(Math.random() * suit.length)]
     }`
-    // 2)
+    // 2) this is our new function (codeception) that is called draw, we have an array named rank and its values. we need to interpoloate it from the array toa string. Math.floor and Math.random is used to round down and produce a whole number. The Math.random also takes the rank array and .lenght methd and tells the code to use all the elements. then suit is called the same way to give a random value from the suit array on line 6.
```

Comment:
You are right about what this is doing but it is just a regular old variable not a function.